terraform/ecs/variables.tf

```diff
@@ -16,4 +16,5 @@ variable "db_passowrd_arn" {type = string}
 variable "db_name_arn" {type = string}
 variable "pub_sub1" {type = string}
 variable "pub_sub2" {type = string}
-variable "target_group_arn" {type = string}
\ No newline at end of file
+variable "target_group_arn" {type = string}
+variable containerimage {type=string}
```

Comment:
is it correct that containerimage don't have quotes on it?

Reply:
yes, all variables on terraform 0.12 or newer don't need ""  or ${}